changelog/15277.txt

```diff
@@ -0,0 +1,11 @@
+```release-note:feature
+**Allows Multiple Issuer Certificates to enable Non-Disruptive
+Intermediate/Root Certificate Rotation**: This introduces /keys and /issuers
+endpoints to allow import, generation and configuration of any number of keys
+or issuers that can be used to issue and revoke certificates.  Keys and Issuers
+can be referred to by (a) a unique UUID; (b) a name; (c) “default”.  If an
+issuer existed prior to this feature, that issuer will be tagged by a migration
+as “default” to allow backwards compatible calls which don’t specify an issuer.
+Creation of new roles will assume an issuer of “default” unless otherwise
+specified.  This default can be configured at /config/issuers and /config/keys.
```

Comment:
This may be too much detail for a changelog.  How to use it should be in the docs.  I'd omit the details about referencing keys and issuers, but add a sentence or two about how having multiple issuers allows in place rotation of CA certificates.